### ISSUE TITLE
Safe Exam Browser version change blocks exam opening

### ISSUE THREAD

**Issue by Spruce8913** [numbas/numbas-lti-provider#334]
When using a .seb file (even a basic one with only [the requirements of the documentation](https://docs.numbas.org.uk/lti/en/latest/admin/lockdown-apps.html#adding-a-safe-exam-browser-settings-file), everything works fine if the students then login with the same SEB version.

However, here's what happens if a student is on a different version (I did not test this holds for every version):
![455521295_353151317732236_4773714929701877111_n](https://github.com/user-attachments/assets/c8c20d7c-ee5e-4d5c-b6c8-682c301b1c07)
The .seb file was created on Windows x64 SEB version 3.7.1, uploaded on NUMBAS LTI provider with the found config key. The student laptop in this case is using Windows x64 SEB version 3.5.0. Note that on the device with SEB 3.7.1 opening NUMBAS exams with .seb config files (and keys) made in older versions of SEB, I do not get this error. Also when I install SEB-windows 3.7.1 on the student laptop, it does work, so it really seems to be a SEB-version issue (despite the config key supposedly being version stable).

The displayed error, judging from [a SEB forum](https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/887), may be related to a configkey mismatch and the lti provider server trying to resupply the .seb file. I also saw [in the release notes for SEB-windows 3.7.1](https://github.com/SafeExamBrowser/seb-win-refactoring/releases) may have had some changes to how the config key is calculated, possibly explaining the issue. I have not tried it yet with other combinations of SEB-windows versions. I also have not tested this on the different SEB versions for mac.

Regardless, this issue would cause quite some issues as in the coming academic year for our students, we do expect many SEB-version older than 3.7.1.

**Comment by Spruce8913** [comment]
I have done some more testing. I originally thought the cause of the issue was the difference between what SEB version was used to create the .seb file and what SEB version was used to open it for the exam. I thus thought I could circumvent the issue by creating the .seb file in an older version than what I expected my students to be using. This didn't solve it.

In the end, I ended up creating 3 .seb files with the same settings but purposely different config key (you can force this by altering the file, saving, removing the alteration and saving again) **in SEB 3.5; 3.6 and 3.7.1 on Windows**. I then proceeded to create separate NUMBAS lti links in our learning platform per SEB version, and tested each out in **in SEB 3.5; 3.6 and 3.7.1 on Windows**. This generates a matrix of 3x3 possibilities of which you can find my draft notes under here:
![458439671_528473666262726_2112936032964328166_n](https://github.com/user-attachments/assets/705a54ae-b726-4fbe-97f6-65195cdc3855)

In summary of the picture above: the SEB version used to create the .seb file does not matter; only the SEB version students use to open the exam matters. On windows, only SEB 3.7.1 (and maybe 3.7.0; didn't test) works. I do not know if this is a bug, since in 3.7.1 (see the link to release notes earlier) there was indeed a correction to how the config key is calculated which I presume is the reason. It's just "unfortunate" in a way that this seemingly means the config key of older versions is considered 'incorrect'...
